addons/mail/models/discuss/discuss_channel.py

```diff
@@ -160,6 +161,17 @@ def _constraint_group_id_channel(self):
 
     # COMPUTE / INVERSE
 
+    @api.depends("channel_member_ids", "auto_recompute_name")
+    def _compute_name(self):
+        for channel in self:
+            if channel.auto_recompute_name:
```

Comment:
I'm not sure if it is applicable for compute of stored fields, but normally compute should assign a value to all records in the set, so there must be a default where channel.name = channel.name or what if there is no member??

Reply:
as I tested, for compute of stored fields, you don't have to do the default, and it works well.